pkgs/development/mobile/androidenv/androidsdk.nix

```diff
@@ -12,12 +12,12 @@ let inherit (stdenv.lib) makeLibraryPath; in
 
 stdenv.mkDerivation rec {
   name = "android-sdk-${version}";
-  version = "25.2.5";
+  version = "26.0.1";
```

Comment:
Seems superseded in current master branch: https://github.com/NixOS/nixpkgs/blob/4120419e10d1edcf9f3835b6e1e3ffe8f285b56d/pkgs/development/mobile/androidenv/generated/packages.nix#L1907-L1920